pkgs/by-name/ig/igir/package.nix

```diff
@@ -39,7 +39,7 @@ buildNpmPackage rec {
     mainProgram = "igir";
     homepage = "https://igir.io";
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ TheBrainScrambler ];
+    maintainers = with maintainers; [ ];
```

Comment:
```suggestion
    maintainers = [ ];
```

Reply:
Let's not be too pedantic in a PR like this.

Also, this overly strict check is subject to removal anyways.